packages/server/hubspot/hubSpotApi.ts

```diff
@@ -0,0 +1,143 @@
+import fetch from 'node-fetch'
```

Comment:
+1 this wasn't introduced in this PR, but the folder is called `hubspot`, but the files within it are spelt `hubSpot`. It'd be nice if they were consistent. Could change the folder name to `hubSpot` to ease my OCD